pkgs/moduleit/entrypoint.nix

```diff
@@ -1,5 +1,6 @@
 { pkgs ? import <nixpkgs> { }
 , configPath
+, pkgs-unstable
```

Comment:
Is this going to be problematic that source-based module compiles need to provide this argument?

Reply:
if they write { pkgs, ... } then the code will still work. I think it's ok.

Reply:
This line will break: https://github.com/replit/nixmodules/blob/9dc9ea9f869597e2a1363a197445165d0aaded61/pkgs/moduleit/moduleit.sh#L13 because it doesn't provide pkgs-unstable. It is okay that it doesn't provide pkgs, because we've provided a default for that.

Potential fix: change the rust module to just use pkgs but override the definition of pkgs for the purposes of the rust module. For example, maybe:
```nix
{
  mkUnstableModule = path: pkgs-unstable.callPackage ../moduleit/entrypoint.nix {
    configPath = path;
  };
}
```

Reply:
Other possible fix: provide a default of pkgs-unstable.

Reply:
I put `import <nixpkgs-unstable> { }` as default for pkgs-unstable. I think this should work as long as you have a channel configured named `nixpkgs-unstable` which they Repls do. Does this work?